lib/active_fedora/file.rb

```diff
@@ -139,6 +145,20 @@ def metadata?
       false
     end
 
+    def external_content?
+      external_url.present?
+    end
+
+    def external_url
+      return nil unless mime_type.start_with? "message/external-body"
+      url = mime_type.split(';').at(2)
+      url.nil? ? nil : url[/\"(.*?)\"/, 1]
```

Comment:
Is it worth pulling in this gem (of 2 commits from 3 years ago) along with its dependency parslet for this?  Maybe it would be better to come up with something good enough that we own instead.